tmt/utils/__init__.py

```diff
@@ -576,7 +576,7 @@ def from_file(
             content = environment_filepath.read_text()
 
         # Parse yaml file
-        if os.path.splitext(filename)[1].lower() in ('.yaml', '.yml'):
+        if filename.rsplit('.')[-1].lower() in ('yaml', 'yml'):
```

Comment:
What about using `Path(filename).suffix` here?

Reply:
Sure, fixed in c23a9b97

Reply:
The suffix includes dots as well, so it should be `.yaml` and `.yml`.
Failing `/tests/core/environment-file` confirms this.

Reply:
Fixed in 0306d54e